src/group_operations/multiplication_operation.jl

```diff
@@ -153,7 +153,7 @@ _doc_diff_left_compose_mult = """
     diff_left_compose!(G::LieGroup{𝔽,<:AbstractMultiplicationGroupOperation}, Y, g, h, X)
 
 Compute the differential of the left group multiplication ``λ_g(h) = g$(_math(:∘))h``,
-which simplifies for an [`AbstractMultiplicationGroupOperation`](@ref) to ``$(_math(:d))λ_g(h)[X] = $(_math(:Ad))(g)[X] = gXg^{-1}``.
+which simplifies for an [`AbstractMultiplicationGroupOperation`](@ref) to ``$(_math(:d))λ_g(h)[X] = $(_math(:Ad))(h^{-1})[X] = h^{-1}Xh``.
```

Comment:
From the interface docs for `diff_left_compose!`:
`dλg(h)[X]=h⁻¹Xh`

Reply:
Oh, thanks for being careful, I have to check on which of the two updates I made the error then.

Reply:
Note: you are comparing against this PR where i updated the docs already. 
Compare against https://juliamanifolds.github.io/LieGroups.jl/previews/PR67/interface/operations

Reply:
Ah, reading your comment carefully it sounds like it is the differential of left compose, that will make the original order correct.

Reply:
I swopped it back to the original order